lib/pbench/test/unit/server/auth/conftest.py

```diff
@@ -1,20 +1,27 @@
+from cryptography.hazmat.primitives.asymmetric import rsa
```

Comment:
'hazmat', huh?  I'm not sure whether to laugh or run screaming.  😵‍💫

Is this statement going to wind us up subject to import/export controls?

Reply:
> 'hazmat', huh? I'm not sure whether to laugh or run screaming.

:laughing: I don't know why `rsa` comes under hazardous material.


> Is this statement going to wind us up subject to import/export controls?

I don't think so but I also don't know what you mean by import/export controls.

Reply:
I thought that the name was very strange/funny until I started to think about it.

> I also don't know what you mean by import/export controls.

Well, for example see the last paragraph on the first page of the Red Hat [Policy on Export Practices](https://source.redhat.com/departments/legal/globallegalcompliance/compliance_folder/red_hat_export_compliance_policypdf).

Reply:
Yeah, I don't know the answer to this one, but I believe this would be true for any mechanism we use to generate the rsa keys, isn't it? Maybe @portante can weigh in?

Reply:
I don't think it is a problem.